sdk/attestation/azure-security-attestation/src/test/java/com/azure/security/attestation/AttestationPolicyManagementTests.java

```diff
@@ -0,0 +1,248 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.security.attestation;
+
+import com.azure.core.http.HttpClient;
+import com.azure.security.attestation.models.AttestationCertificateManagementBody;
+import com.azure.security.attestation.models.JsonWebKey;
+import com.azure.security.attestation.models.PolicyCertificatesModifyResponse;
+import com.azure.security.attestation.models.PolicyCertificatesResponse;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.JWSAlgorithm;
+import com.nimbusds.jose.JWSHeader;
+import com.nimbusds.jose.JWSObject;
+import com.nimbusds.jose.JWSSigner;
+import com.nimbusds.jose.Payload;
+import com.nimbusds.jwt.JWTClaimsSet;
+import net.minidev.json.JSONArray;
+import net.minidev.json.JSONObject;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import reactor.test.StepVerifier;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.text.ParseException;
+import java.util.Collections;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class AttestationPolicyManagementTests extends AttestationClientTestBase {
+    private static final String DISPLAY_NAME_WITH_ARGUMENTS = "{displayName} with [{arguments}]";
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("getAttestationClients")
+    void testGetPolicyManagementCertificates(HttpClient httpClient, String clientUri)
+        throws ParseException {
+
+        AttestationClientBuilder attestationBuilder = getBuilder(httpClient, clientUri);
+        PolicyCertificatesClient client = attestationBuilder.buildPolicyCertificatesClient();
+
+        PolicyCertificatesResponse response = client.get();
+        JWTClaimsSet claims = verifyAttestationToken(httpClient, clientUri, response.getToken()).block();
```

Comment:
You can replace these last three lines with:

```java
StepVerifier.create(verifyAttestationToken(httpClient, clientUri, response.getToken())
    .assertNext(claims -> verifyGetPolicyCertificatesResponse(clientUri, claims))
    .expectComplete()
    .verify();
```

Reply:
QQ: What is the difference between the two constructs:

```
.verifyComplete();
```

and
```
.expectComplete()
.verify()
```
?? They seem to be the same, is there a reason one is preferred over the other?

Reply:
I like being explicit but they're the same thing. :) No preference